docs/pages/deployment/monitoring.rst

```diff
@@ -108,6 +113,8 @@ Explanation of ambiguous/complex entries in the diagnostics:
 * ``vcr.issuer.issued_credentials_count`` holds the total number of credentials issued by the local node
 * ``vcr.issuer.revoked_credentials_count`` holds the total number of revoked credentials issued by the local node
 * ``vcr.verifier.revocations_count`` holds the total number of revoked credentials (public and private VCs)
+* ``vdr.conflicted_did_documents.total_count`` holds the total number of DID documents that have parallel updates. This may indicate a stolen private key
```

Comment:
```suggestion
* ``vdr.conflicted_did_documents.total_count`` holds the total number of DID documents that are conflicted (have parallel updates). This may indicate a stolen private key
```